queries.sql

```diff
@@ -122,4 +122,63 @@ AND A.escape_attempts = 0
 
 SELECT id_owners, count(*) FROM animals 
     GROUP BY id_owners
-    HAVING COUNT(*)>2
\ No newline at end of file
+    HAVING COUNT(*)>2
+
+SELECT animals.name, visits.date_visit
+FROM visits
+LEFT JOIN animals ON animals.id_animals =  visits.id_animal
+LEFT JOIN vets ON vets.id_vets = visits.id_animal
+WHERE vets.name_vets = 'William Tatcher'
+ORDER BY visits.date_visit DESC
+LIMIT 1;
+
+SELECT DISTINCT animals.name
+FROM visits
+LEFT JOIN animals ON animals.id_animals = visits.id_animal
+LEFT JOIN vets ON vets.id_vets = visits.id_vet
+WHERE vets.name_vets = 'Stephanie Mendez';
+
+SELECT vets.*, species.name
+FROM vets
+LEFT JOIN specializations ON vets.id_vets = specializations.id_vet
+LEFT JOIN species ON specializations.id_species = species.id_species;
+
+SELECT animals.name, visits.date_visit
+FROM visits
+LEFT JOIN animals ON animals.id_animals = visits.id_animal
+LEFT JOIN vets ON vets.id_vets = visits.id_vet
+WHERE vets.name_vets = 'Stephanie Mendez'
+AND date_visit > '2020-04-01' AND date_visit < '2020-08-30';
+
+SELECT A.name, COUNT(A.name)
+FROM visits
+LEFT JOIN animals A ON A.id_animals = visits.id_animal
+ORDER BY COUNT(A.name) DESC
+
+SELECT animals.name, visits.date_visit
+FROM visits
+LEFT JOIN animals ON animals.id_animals = visits.id_animal
+LEFT JOIN vets ON vets.id_vets = visits.id_vet
+ORDER BY visits.date_visit ASC
+LIMIT 1;
```

Comment:
Please add a query to find Maisy Smith's first visit name?

Reply:
Hey, I appreciate your effort in trying to do your work but I already have that part.

![image](https://user-images.githubusercontent.com/74747182/149394600-a5093bbd-ea39-431b-a797-eeff55124063.png)
 Can you please double-check and leave a comment for the next reviewer so he can just approve my work? thanks in advance!!!